src/dotnet/APIView/APIViewWeb/Models/ReviewRevisionModel.cs

```diff
@@ -27,5 +27,7 @@ public string Name
 
         [JsonIgnore]
         public ReviewCodeFileModel SingleFile => Files.Single();
+
+        public string Uploader { get; set; }
```

Comment:
Do we use uploader in other places? I think it's `Author` on the review.

Reply:
Fixed